tests/test_pathlib.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+"""
+This tests timezone-aware date/time objects, which need pytz. Naive date/times should
+work with just Python code functionality, and are tested in `nonp`.
+"""
+
+from pathlib import Path
+from json_tricks import dumps, loads
+from json_tricks.utils import is_py3
+
+
+PATHS = [
+		Path(),
+		Path(__file__),
```

Comment:
I think it's better to use a fixed path, so that the test is the same on each system

Maybe different OS paths too?

Reply:
Done.